manim/constants.py

```diff
@@ -259,6 +261,10 @@
 DEFAULT_QUALITY: str = "high_quality"
 DEFAULT_QUALITY_SHORT = QUALITIES[DEFAULT_QUALITY]["flag"]
 
+DEFAULT_RENDER_NOTE: str = (
```

Comment:
I don't think this is worth having as a separate global constant (keep in mind that these get imported into the local namespace of everyone doing `from manim import *`). The note itself is fine, I'd just rather see it added to the extended EPILOG string directly.

Reply:
ok, I'll fix that.